volmaster/daemon.go

```diff
@@ -95,7 +96,7 @@ func (d daemonConfig) handleRemove(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := removeVolume(vc); err != nil {
+	if err := removeVolume(vc, d.timeout); err != nil {
```

Comment:
I'm not opposed to this, but now is probably out of scope; I'd like to do this with a separate patch. I'll put in a ticket (take it if you want to).